mediawiki/LocalSettings.d/LocalSettings.override.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+error_reporting( -1 );
```

Comment:
do not put to main config. maybe a dev override that is only present in the dev container comes in handy.